src/Tizen.NUI/src/internal/Interop/Interop.TextEditor.cs

```diff
@@ -316,6 +316,15 @@ internal static partial class TextEditor
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_InputFilteredSignal")]
             public static extern global::System.IntPtr InputFilteredSignal(global::System.Runtime.InteropServices.HandleRef textEditorRef);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_Property_CopyText")]
```

Comment:
Have you tested this patch to work?

EntryPoint = "CSharp_Dali_TextEditor_Property_CopyText" is different from function name appended to dali-csharp-binder.
So I get a System.EntryPointNotFoundException.
```
dali-csharp-binder :
SWIGEXPORT char* SWIGSTDCALL CSharp_Dali_TextEditor_CopyText(void *pTextEditor)
```

+ Other functions have the same problem.

Reply:
Hello,
Sorry not yet.
I was trying to prepare the environment for the full cycle test but have not reached to this exception yet.
I will confirm once the full cycle test is done.
Thank you for understanding.

Reply:
@wonrst 
Tested & Verified on NUI emulator.
I have tested the apis on Tizen.NUI.Samples using 5 buttons and 2 text-editors.
2 buttons to call SelectText in the different editors and the 3 other buttons to copy, cut and paste.
The manual tests on NUI work as expected.
Thanks!

Reply:
I also checked that it works fine. Please resolve conflicts.